### ISSUE TITLE
How do I extract the required information from the nii.gz tag file?

### ISSUE THREAD

**Issue by tanjia123456** [deepmedic/deepmedic#210]
Hello, I'm going to add a loss function. I found that you used the cross entropy loss function.
# p_ y_ given_ x_ train : tensor5 [batchSize, classes, r, c, z]
# y: T.itensor4('y'). Dimensions [batchSize, r, c, z]
So I have a question, I already have the marked segmentation file, but how can I extract y_GT from .nii file? I want to extract y_Gt 'dimension is related to P_ y_ given_ x_train
